camunda-formio-tasklist-vue/src/components/mixins/TaskListMixin.vue

```diff
@@ -8,5 +8,6 @@ import BaseMixin from './BaseMixin.vue';
 export default class TaskListMixin extends Mixins(BaseMixin) {
 
   public maximize = true;
+  public userName = '';
 }
```

Comment:
try to maintain the variable format like "public/private userName: <type> = defaultvalue